src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobDispatcherTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.blobrouter.services.storage;
+
+import com.azure.core.test.TestBase;
+import com.azure.storage.blob.BlobServiceClient;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.blobrouter.util.StorageClientsHelper;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+class BlobDispatcherTest extends TestBase {
```

Comment:
Test missing for exception scenario ?

Reply:
Just logging in exception never mind

Reply:
Added anyway. More complete coverage and in case this edge case will change with like error notification ping - will have to be addressed